apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/ISpan.cs

```diff
@@ -45,8 +47,10 @@ namespace Google.Cloud.Diagnostics.Common
     ///     t2.Start();
     /// }
     /// </code>
```

Comment:
Not sure if the code tags need to be in or out of the para tag. You may just want to double-check the help that gets generated.